src/SqlSerializer.h

```diff
@@ -74,25 +75,26 @@ class SqlSerializer {
 
     sqlite3_stmt *populate_insert_argument_statement(const closure_info_t &info,
                                                      int index);
+
     bool verbose;
     int indentation;
-    sqlite3 *database = nullptr;
+    sqlite3 *database = NULL;
     std::ofstream trace;
-    sqlite3_stmt *insert_metadata_statement = nullptr;
-    sqlite3_stmt *insert_function_statement = nullptr;
-    sqlite3_stmt *insert_argument_statement = nullptr;
-    sqlite3_stmt *insert_call_statement = nullptr;
-    sqlite3_stmt *insert_promise_statement = nullptr;
-    sqlite3_stmt *insert_promise_association_statement = nullptr;
-    sqlite3_stmt *insert_promise_evaluation_statement = nullptr;
-    sqlite3_stmt *insert_promise_return_statement = nullptr;
-    sqlite3_stmt *insert_promise_lifecycle_statement = nullptr;
-    sqlite3_stmt *insert_promise_argument_type_statement = nullptr;
-    sqlite3_stmt *insert_gc_trigger_statement = nullptr;
-    sqlite3_stmt *insert_type_distribution_statement = nullptr;
-    sqlite3_stmt *insert_environment_statement = nullptr;
-    sqlite3_stmt *insert_variable_statement = nullptr;
-    sqlite3_stmt *insert_variable_action_statement = nullptr;
+    sqlite3_stmt *insert_metadata_statement = NULL;
```

Comment:
nullptr vs NULL - see other pull request

Reply:
Not a strong reason except that nulllptr makes more sense in the C++ land. `sqlite3_stmt` will be exclusively handled by sqlite3 which is purely C.

Reply:
Then I think it is a good idea to leave them as nullptr and get the type checking benefit when we do ocasionally use them in C++ land.